docs/data/joy/components/modal/modal.md

```diff
@@ -8,113 +8,139 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/
 
 # Modal
 
-<p class="description">The modal component provides a solid foundation for creating dialogs, popovers, lightboxes, or whatever else.</p>
-
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+<p class="description">The modal component provides a solid foundation for creating dialogs, popovers, lightboxes, and other interactive elements.</p>
 
 ## Introduction
 
-Joy UI provides three modal-related components:
+Modals are interactive dialog boxes or overlays that capture user inputs or present information.
+
+In Joy UI, modals are managed through three related components:
 
 - [`Modal`](#basic-usage): A container that renders its `children` node in front of a backdrop component.
```

Comment:
We can drop the `backticks` from these list items (same suggestion goes for the other two below this one) - we only need to use `<ComponentCase />` (styled like a React component) when it'd be unclear otherwise

```suggestion
- [Modal](#basic-usage): A container that renders its `children` node in front of a backdrop component.
```